Validation/RecoParticleFlow/python/particleFlowDQM_cff.py

```diff
@@ -90,9 +107,10 @@ def createGenJetPlots(ptbins, etabins):
 
     jetResponseDir = cms.vstring( vjetResponseDir ),
     genjetDir = cms.string( genjetDir ),
+    offsetDir = cms.string( offsetDir ),
     ptBins = cms.vdouble( ptbins ),
     etaBins = cms.vdouble( etabins ),
-    recoPtCut = cms.double( 15. )
+    recoPtCut = cms.double( 10. )
 
```

Comment:
Please remove thi empty line, for an easier reading

Reply:
Done.